# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/sync15/src/client/mod.rs
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf third_party/rust/sync15/src/client/mod.rs
--- a/third_party/rust/sync15/src/client/mod.rs	Wed Mar 01 05:53:30 2023 +0000
+++ b/third_party/rust/sync15/src/client/mod.rs	Wed Mar 01 17:52:01 2023 -0500
@@ -25,7 +25,7 @@
 mod token;
 mod util;
 
-pub(crate) use coll_state::{CollState, LocalCollStateMachine};
+pub(crate) use coll_state::CollState;
 pub(crate) use coll_update::{fetch_incoming, CollectionUpdate};
 pub(crate) use collection_keys::CollectionKeys;
 pub(crate) use request::InfoConfiguration;